# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/ctwm/yyparse.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

*** ./gram.y~	Tue Dec 11 16:38:52 2001
--- ./gram.y	Thu Jun 13 08:42:58 2002
***************
*** 419,425 ****
  		| WINDOW_RING		{ list = &Scr->WindowRingL; }
  		  win_list
  		| WINDOW_RING           { Scr->WindowRingAll = TRUE; }
- 		;
  		| WINDOW_RING_EXCLUDE    { if (!Scr->WindowRingL)
  		                              Scr->WindowRingAll = TRUE;
  		                          list = &Scr->WindowRingExcludeL; }
--- 419,424 ----
***************
*** 427,432 ****
--- 426,432 ----
  
  		| WINDOW_GEOMETRIES 	 {  }
  		  wingeom_list
+ 		;
  
  noarg		: KEYWORD		{ if (!do_single_keyword ($1)) {
  					    twmrc_error_prefix();
***************
*** 683,691 ****
  		| wingeom_entries wingeom_entry
  		;
  
! wingeom_entry	: string string	{ AddToList (&Scr->WindowGeometries, $1, $2) }
! 
! 
  
  squeeze		: SQUEEZE_TITLE { 
  				    if (HasShape) Scr->SqueezeTitle = TRUE;
--- 683,690 ----
  		| wingeom_entries wingeom_entry
  		;
  
! wingeom_entry	: string string	{ AddToList (&Scr->WindowGeometries, $1, $2); }
! 		;
  
  squeeze		: SQUEEZE_TITLE { 
  				    if (HasShape) Scr->SqueezeTitle = TRUE;
***************
*** 834,839 ****
--- 833,839 ----
  occupy_workspc_entry	: string {
  				AddToClientsList ($1, client);
  			  }
+ 			;
  
  occupy_window_list	: LB occupy_window_entries RB {}
  			;
***************
*** 845,850 ****
--- 845,851 ----
  occupy_window_entry	: string {
  				AddToClientsList (workspace, $1);
  			  }
+ 			;
  
  icon_list	: LB icon_entries RB {}
  		;
***************
*** 962,967 ****
--- 963,970 ----
  					  RemoveDQuote(ptr);
  					  $$ = (unsigned char*)ptr;
  					}
+ 		;
+ 
  number		: NUMBER		{ $$ = $1; }
  		;
  
